data/outfits.txt

```diff
@@ -60,6 +60,7 @@ outfit "D14-RN Shield Generator"
 outfit "Skirmisher Shield Generator"
 	category "Systems"
 	cost 20000
+	thumbnail "outfit/skirmisher_shield"
```

Comment:
The underscore should be a space, as with all other outfit sprite names.

Also, it's pretty obvious that this is just a recolor of the regular D23-QP...